pkg/notify/subscriptionnotification_test.go

```diff
@@ -30,7 +30,7 @@ func buildConfig() (config.SubscriptionConfig, error) {
 	props.AddStringProperty("central.subscriptions.notifications.smtp.username", "bill", "")
 	props.AddStringProperty("central.subscriptions.notifications.smtp.password", "pwd", "")
 	props.AddStringProperty("central.subscriptions.notifications.smtp.subscribe.subject", "subscribe subject", "")
-	props.AddStringProperty("central.subscriptions.notifications.smtp.subscribe.body", "subscribe body", "")
+	props.AddStringProperty("central.subscriptions.notifications.smtp.subscribe.body", "Subscription created for Catalog Item:  <a href= ${catalogItemUrl}> ${catalogItemName} ${catalogItemId}</a> <br/>", "")
```

Comment:
this should be updated for the new template variable

Reply:
Yes, the plan was me to finish the unit tests today